docs/dev/checkstyle/checkstyle.md

```diff
@@ -0,0 +1,47 @@
+# Checkstyle
+
+## License Headers with IntelliJ
```

Comment:
You will on your onboarding, and there's where you'd discover that link or skip it. It's not really docs on how to work with Checkstyle, because Checkstyle will work with you, via CI. It's about how to setup your IJ for easier working with this repo

Reply:
Moved to an IJ.md file